tensorflow/core/kernels/eigen_mkldnn_contraction_kernel_test.cc

```diff
@@ -13,6 +13,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+// Need to #include Eigen's Tensor class first because Eigen/CXX11/FixedPoint
+// depends on the file but doesn't include it. This breaks compilation on
+// clang.
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
 #include "third_party/eigen3/unsupported/Eigen/CXX11/FixedPoint"
```

Comment:
Could you please put clang-format off around FixedPoint input (https://github.com/tensorflow/tensorflow/blob/7817237d5852c7778d3ba03c40f139e6f2c37a76/tensorflow/core/framework/numeric_types.h#L21-L25). I think it might get messed up during copybara import-export.

Reply:
Good idea. I will make that change and verify that `clang-format -style=Google` doesn't change the file.